server/rpc.go

```diff
@@ -217,6 +220,39 @@ func verifySegCreds(orch Orchestrator, segCreds string) (*core.SegTranscodingMet
 	return md, nil
 }
 
+func genPayment(sess *BroadcastSession) (string, error) {
+	if sess.Sender == nil {
+		return "", nil
+	}
+
+	ticket, seed, sig, err := sess.Sender.CreateTicket(sess.PMSessionID)
+	if err != nil {
+		return "", err
+	}
+
+	protoTicket := &net.Ticket{
+		Recipient:         ticket.Recipient.Bytes(),
+		Sender:            ticket.Sender.Bytes(),
+		FaceValue:         ticket.FaceValue.Bytes(),
+		WinProb:           ticket.WinProb.Bytes(),
+		SenderNonce:       ticket.SenderNonce,
+		RecipientRandHash: ticket.RecipientRandHash.Bytes(),
+	}
+	protoPayment := &net.Payment{
+		Ticket: protoTicket,
+		Sig:    sig,
+		Seed:   seed.Bytes(),
+	}
+
+	data, err := proto.Marshal(protoPayment)
+	if err != nil {
+		glog.Error("Unable to marshal ", err)
```

Comment:
why do we log this error specifically whereas with other errors we return with no logging?

Reply:
I forgot to remove this log - I don't think it is necessary because we wrap the error returned by `genPayment()` and log it [here](https://github.com/livepeer/go-livepeer/pull/662/files#diff-44bb03d888e11d6429223a7b0e55220dR477)